lib/manager/cocoapods/artifacts.ts

```diff
@@ -78,16 +78,13 @@ export async function updateArtifacts(
     ) || { groups: { cocoapodsVersion: 'latest' } };
     cocoapodsVersion = match.groups.cocoapodsVersion;
 
-    cmdParts.push(
-      ...[
-        'docker',
-        'run',
-        '--rm',
-        `-v ${cwd}:${cwd}`,
-        `-w ${cwd}`,
-        `renovate/cocoapods:${cocoapodsVersion} pod`,
-      ]
-    );
+    cmdParts.push('docker run --rm');
+    if (config.dockerUser) {
+      cmdParts.push(`--user=${config.dockerUser}`);
+    }
+    cmdParts.push(`-v ${cwd}:${cwd}`);
+    cmdParts.push(`-w ${cwd}`);
+    cmdParts.push(`renovate/cocoapods:${cocoapodsVersion} pod`);
```

Comment:
I left this multiline pushes since most probably we'll abstract docker logic somehow, so it will be simpler to modify

Reply:
`pod` can be moved to common step outside the if/else statement since all `cmdParts` are just joined with a space